packages/devtools_app/lib/src/memory/memory_chart.dart

```diff
@@ -922,22 +1002,20 @@ class SelectedDataPoint extends LineChartMarker {
   final int _titlesAndroidLines = _titlesAndroid.split('\n').length;
 
   // These are the alpha blended values.
-  final List<Color> _androidColors = [
+  final List<LegendColor> _androidColors = [
     // TODO(terry): Dash crashes Canvas unimplemented in Flutter Web
     //              use a green color on Web see Flutter issue/49882.
```

Comment:
dashes are now supported. Remove the kIsWeb checks for dash support.
https://github.com/flutter/flutter/issues/48386

Reply:
Done.